lib/boss/config.rb

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ruby
+
```

Comment:
What's this file? There seems to be references to AMQP and EventMachine. And also bin/boss has it's own config parsing and this file is only required in bin/boss_clean_errors

Reply:
Correct - I found the utilities were not DRY so I pulled out the config into a module.
It probably does too much like creating a Storage and running when required rather than being a Class
It does make writing the simple tools easier but I didn't integrate it with main boss yet.
You're right about the EM code - I must have missed some patches for that.

Reply:
When I think about it - the utilities don't need to access AMQP at all. They just need the storage...